litr/src/main/java/com/linkedin/android/litr/thumbnails/queue/PriorityThreadExecutor.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2021 LinkedIn Corporation
+ * All Rights Reserved.
+ *
+ * Licensed under the BSD 2-Clause License (the "License").  See License in the project root for
+ * license information.
+ */
+package com.linkedin.android.litr.thumbnails.queue
+
+import java.util.concurrent.*
+
+/**
+ * Helpers for constructing an [ExecutorService] backed by a [PriorityBlockingQueue] for prioritizing tasks.
+ */
+object PriorityExecutorUtil {
+    fun newSingleThreadPoolPriorityExecutor() = ThreadPoolExecutor(
+        1, 1, 0L,
```

Comment:
nip: these should probably be on separate lines, they seem to be all over the place.